src/models/peer.rs

```diff
@@ -312,30 +307,26 @@ impl PeerStanding {
     /// Sanction peer and return latest standing score
```

Comment:
With your changed function signature of  `sanction`, this comment is no longer correct.

Reply:
Good point. :+1: